### ISSUE TITLE
Older Maven expressions are deprecated and need to be replaced

### ISSUE THREAD

**Issue by eeverman** [eeverman/andhow#546]
The Maven build is generating warning like this:
```
[WARNING] The expression ${groupId} is deprecated. Please use ${project.groupId} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
```

These issues need to be fixed in the pom files in the project.  This is a very simple fix if you are familiar with Maven at all.

**Comment by alex-kar** [comment]
Seems like only `${project.groupId}` and `${project.version}` are used in the project. Didn't find any usage of `${groupId}` or `${groupId}`.

**Comment by eeverman** [comment]
@alex-kar  If you do a global text search on the project, those deprecated expressions show up three times.

**Comment by alex-kar** [comment]
@eeverman I've been searching on wrong branch. Can I take it?

**Comment by eeverman** [comment]
@alex-kar Its yours - Thank you!

**Comment by eeverman** [comment]
Fixed by #551.
Thanks @alex-kar!
